docs/guides/cluster.mdx

```diff
@@ -22,20 +22,29 @@ This guide shows how to deploy a distributed Restate cluster consisting of 3 nod
 To deploy a 3 node distributed Restate cluster, copy the `docker-compose.yml` and run `docker compose up`.
 
 ```yaml
+# Common environment variables for all restate nodes
 x-environment: &default-env
+  # To customize logging, check https://docs.restate.dev/operate/monitoring/logging
+  RESTATE_LOG_FILTER: "restate=info"
+  # Cluster name, this will be visible in few places such as logs
   RESTATE_CLUSTER_NAME: "my-cluster"
+  # In this setup every node fulfills every role.
+  # For more info on disaggregated setups, check https://docs.restate.dev/deploy/server/cluster/
   RESTATE_ROLES: '["admin","worker","log-server","metadata-server"]'
-  RESTATE_LOG_FILTER: "restate=info"
-  RESTATE_AUTO_PROVISION: "false"
+  # Internal metadata and log server configuration
   RESTATE_BIFROST__DEFAULT_PROVIDER: "replicated"
   RESTATE_BIFROST__REPLICATED_LOGLET__DEFAULT_LOG_REPLICATION: 2
   RESTATE_METADATA_SERVER__TYPE: "replicated"
-  RESTATE_METADATA_CLIENT__ADDRESSES: '["http://node1:5122","http://node2:5122","http://node3:5122"]'
+  # This needs to be configured with the hostnames/ports the nodes can use to talk to each other.
+  # In this setup, they interact within the "internal" Docker compose network setup.
+  RESTATE_METADATA_CLIENT__ADDRESSES: '["http://restate-1:5122","http://restate-2:5122","http://restate-3:5122"]'
```

Comment:
Curious why do you prefer `restate-N` over `nodeN`? The compose project is already called `restate`, it seems more natural to have containers named `restate/node-N` than `restate/restate-N`.

Reply:
in case you wanna copy paste in this config other containers, such as kafka